nin/backend/nin

```diff
@@ -15,7 +16,7 @@ program
   .action(function(project){
     project = path.resolve(process.cwd(), project || '');
     console.log('compiling project');
-    compile.compile(project);
+    compile.compile(utils.findProjectRootOrExit(project));
```

Comment:
How does this automatic root finding play together with the optional `project` argument?